wiki/Rules/Rules (Greek)

```diff
@@ -0,0 +1,80 @@
+Κανόνες
```

Comment:
Change the file name to `el.md`.

Reply:
Alright done. I remember last time someone had told me to mention in the title that it's about the Rules because "el.md" can be very vague.